symphony-bdk-core/src/main/java/com/symphony/bdk/core/activity/command/CommandActivity.java

```diff
@@ -0,0 +1,66 @@
+package com.symphony.bdk.core.activity.command;
+
+import static com.symphony.bdk.core.service.datafeed.util.RealTimeEventsBinder.bindOnMessageSent;
+
+import com.symphony.bdk.core.activity.exception.ActivityExecutionException;
+import com.symphony.bdk.core.activity.AbstractActivity;
+import com.symphony.bdk.core.service.datafeed.RealTimeEventListener;
+import com.symphony.bdk.gen.api.model.V4MessageSent;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.apiguardian.api.API;
+import org.w3c.dom.Document;
+import org.xml.sax.SAXException;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.function.Consumer;
+
+import javax.annotation.Nonnull;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+
+/**
+ * A form reply activity corresponds to any message send in a chat where the bot is part of.
+ */
+@Slf4j
+@API(status = API.Status.STABLE)
+public abstract class CommandActivity<C extends CommandContext> extends AbstractActivity<V4MessageSent, C> {
+
+  /** Used to parse PresentationML message */
+  private static final DocumentBuilder DOCUMENT_BUILDER = initDocumentBuilder();
+
+  /** The bot display name */
+  @Getter @Setter private String botDisplayName;
+
+  /** {@inheritDoc} */
+  @Override
+  public void bindToRealTimeEventsSource(Consumer<RealTimeEventListener> realTimeEventsSource) {
```

Comment:
It should be `protected`, no?

Reply:
Eagle eyes! 
![](https://media.giphy.com/media/RKw94AILIkuWY/source.gif)

Reply:
Fixed